lib/orogen/spec/deployment.rb

```diff
@@ -379,17 +379,28 @@ def periodic(value)
 
             # Marks this task as being explicitely triggered (the default). To
             # make it periodic, call #period with the required period
-            def triggered
+            def triggered(timeout: nil)
```

Comment:
is time always in nanosec in _tools/orogen_? If not, documenting the `timeout` parameter would be nice

Reply:
I second the need for documenting the unit, here and also for periodic. As far as i can tell, both should be seconds. (`setAperiodicTriggerTimeout` takes nanoseconds, because the system calls take nanoseconds, factor 1e9 relative to seconds.)